content/applications/hr/payroll/work_entry_analysis.rst

```diff
@@ -0,0 +1,78 @@
+===================
+Work entry analysis
+===================
+
+The default *Work Entries Analysis* report provides an overview of the validated work entries for
+the current month. To view this report, navigate to :menuselection:`Payroll app --> Reporting -->
+Work Entry Analysis`.
+
+The work entries appear in a pivot table, with the default filters of :guilabel:`Current month:
+(Month)(Year)` and :guilabel:`Validated`. The various types of :doc:`work_entries` populate the
+rows, while the :guilabel:`Total` values populate the only visible column.
+
+To change the displayed information, click :icon:`fa-plus-square` :guilabel:`Total` above the main
+column, revealing a drop-down menu of available metrics. Click on one of the available groupings,
+and the data is further organized by that selected metric. The default options are :guilabel:`Work
+Entry Type`, :guilabel:`Employee`, and :guilabel:`Department`. If in a multi-company database, a
+:guilabel:`Company` option also appears.
+
+Work entry analysis comparison
+==============================
+
+It is possible to compare the work entries from one time period to a previous time period. To view
+this comparison, first navigate to :menuselection:`Payroll app --> Reporting --> Work Entry
+Analysis`.
+
+Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, revealing a
+drop-down menu. Under the :icon:`fa-adjust` :guilabel:`Comparison` section, click on either
```

Comment:
Ah! It showed up after I selected a few months first:
![Screenshot 2024-10-08 at 4 00 54 PM](https://github.com/user-attachments/assets/08230337-30b1-4277-bb47-a26bd6154912)

It appears you need to select a time range first, in order to see comparison period options. Can you validate this and update the content to reflect the most accurate steps?

Also, it seems that in order to recreate your screenshot, the X-Axis group by needs to be explicitly declared: click **Total ➕ > Date created > Months**, no?

Reply:
OK, I double checked on 3 runbots (all 17.0) and they all show me the comparison, by default:
![image](https://github.com/user-attachments/assets/8d3c2cdd-e235-4142-add1-0307569a045c)

The default filter is the current month- so if you don' have any information showing, that could be why it is not appearing. There has to be some data for it to show. Do you know if you had a blank/empty table when you didn't see it?

When I first was looking to see when the comparison appears, because I had issues in another report, you only need one specific date selected to make it appear. So if you were to remove ALL filters, and you select '2024', the comparison appears.  So as long as any time period is selected, whether it's a month, year, or quarter then it appears - since it will compare the data with the previous of whatever is selected (quarter, month, etc).

Reply:
I also double-checked the X-axis question, and the default displays it like in my screenshot, but if you select something else or make changes to the default, it might not appear like that. So I think I can either add a note that states that, and that should clear things up!

Reply:
OK, the note was updated to be more clear. LMK if you still have any issues- but you should be good!

Reply:
I just confirmed that if you follow the instructions on line 59 exactly, and _don't_ remove the default filters when landing on the dashboard (like I did earlier), then, indeed, the comparison options show up.

It does seem like it won't show up unless a time window is explicitly selected. Maybe worth adding as a **Note**, yea...since removing default filters is almost a knee-jerk reaction for anyone who regularly runs reports (you instead just go right to custom and build your own logic, usually)